api.md

```diff
@@ -1453,20 +1453,28 @@ ALTER TABLE metrics SET (timescaledb.compress, timescaledb.compress_orderby = 't
 
 ## add_compression_policy() :community_function: [](add_compression_policy)
 Allows you to set a policy by which the system will compress a chunk
-automatically in the background after it reaches a given age.
+automatically in the background after it reaches a given age. Note that
+a hypertable must have the `timescaledb.compress` storage parameter set
+in order to be compressed.
```

Comment:
This instruction feels confusing, at it refers to a specific "storage parameter", which (a) isn't in _this_ API call, and (b) isn't in the `create_hypertable` call.

I guess I understand that when users use `ALTER TABLE` to set a compression policy, this "storage parameter" gets set, but I wonder if this is too "in the weeds" and instead we should uplevel it.

```suggestion
automatically in the background after it reaches a given age. 

Note that compression policies can only be created on hypertables that already
have compression enabled, e.g., via the [`ALTER TABLE`](/api#compression_alter-table) command
to set `timescaledb.compress` and other configuration parameters.
```